daisy/workflow/workflow.go

```diff
@@ -343,6 +381,27 @@ func (w *Workflow) nameToInstanceLink(n string) string {
 	return fmt.Sprintf("projects/%s/zones/%s/instances/%s", w.Project, w.Zone, n)
 }
 
+type gcsLogger struct {
```

Comment:
:P I usually place functions right above where they are called to appease the entropy gods. Moved this to the top to meet you absurd ordering ideas